agent/03_agent_deployment.sh

```diff
@@ -107,17 +119,70 @@ stringData:
   .dockerconfigjson: '${pull_secret}'
 
 EOF
+
+    # Get the generated mac addresses to set up nmstate
+    mac_list=($(sudo virsh net-dumpxml ${BAREMETAL_NETWORK_NAME} | grep -Po '(?<=host mac=).*' | cut -d' ' -f1))
+    ip_list=$2
+
+    add_host_dns_entries ${ip_list}
+
+    num_ips=${#ip_list[@]}
+
+    echo -n "" > ${FLEETING_MANIFESTS_PATH}/nmstateconfig.yaml
```

Comment:
Is this effectively required?

Reply:
Since the cat now appends to the file I wanted to make sure the file doesn't or is empty, but it looks like 'make clean' removes all of the manifest files so yes I can remove this line.